robot/install.bash

```diff
@@ -1,4 +1,23 @@
-#! /usr/bin/env bash
+#!/bin/bash
+
+if [[ "$USER" != "amigo" ]]
+  then
+    read -r -p "This package ($target) should only be installed on the robot itself, not on your own PC. Are you sure to continue? [y/N] " response
+    response=${response,,}    # tolower
+    if [[ "$response" =~ ^(yes|y)$ ]]
+    then
+      :
+    else
+      return 1
+    fi
+fi
```

Comment:
I use amigo myself, plus on HERO1 we use the `administrator` account. So I think we skip this as well.

Reply:
So here you don't want protection against idiots? :wink:

Reply:
If people want to destroy their own machine, I don't care 🤣